src/layout/header/DonationButton.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { DollarOutlined } from '@ant-design/icons'
+import { Box, Grid, Modal, Typography } from '@mui/material'
+
+const style = {
+  position: 'fixed',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  maxWidth: 1000,
+  width: '90%',
+  bgcolor: 'background.paper',
+  boxShadow: 24,
+  p: 4,
+} as const
+
+export const DonationButton = () => {
+  const { t } = useTranslation()
+  const [open, setOpen] = useState(false)
+
+  const tooltip_title = t('donate_title')
+
+  const onClose = () => setOpen(false)
+  const onOpen = () => setOpen(true)
+
+  return (
+    <>
+      <button
+        className="theme-icon"
+        onClick={onOpen}
+        aria-label={tooltip_title}
+        title={tooltip_title}>
+        <DollarOutlined style={{ fontSize: '1.5em' }} />
+      </button>
+      <Modal
+        open={open}
+        onClose={onClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description">
+        <Box sx={style}>
```

Comment:
```suggestion
        <Box dir={i18n.dir()} sx={style}>
```
It could be better to do something similar to the above instead of the index.html change